### ISSUE TITLE
Recalculation of NOW()

### ISSUE THREAD

**Issue by MHOOO** [gristlabs/grist-core#372]
I'm currently using grist-core for time tracking. To display the amount of time that I still need to work on a single day, I am using the `NOW()` function to get the current time. Then I subtract the start time from that. However, cells which use that function only get updated sporadically. I've seen anything from 10 to 40 minutes.
Is there some way of forcing a recalculation of `NOW()`?

**Comment by paulfitz** [comment]
Hi @MHOOO, sorry for the delay responding. `NOW()` is recalculated at hourly intervals. This interval is currently not configurable - it is set here: https://github.com/gristlabs/grist-core/blob/9451fb9597ddf5aa1c464f887dbc672439aefa18/app/server/lib/ActiveDoc.ts#L151-L152

There isn't a convenient way of forcing an update that I know of. You could click on the :skull_and_crossbones: in document settings to force a complete document reload, which would do it, but that isn't handy.

**Comment by MHOOO** [comment]
No worries! I've also been on holidays.
Is this something that should be configurable, e.g. on a documents setting page or via env vars?

**Comment by paulfitz** [comment]
Yes, ideally this would be configurable. Google sheets offers to do recalculation:
 * On change
 * On change and every minute
 * On change and every hour

**Comment by MHOOO** [comment]
Are there any plans on having a document settings dialog or would environment variables suffice?
